dashboard/src/components/AppView/ChartInfo/ChartUpdateInfo.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  checkCircleIcon,
+  circleArrowIcon,
+  ClarityIcons,
+  exclamationTriangleIcon,
+} from "@clr/core/icon-shapes";
+import { CdsIcon } from "components/Clarity/clarity";
+import * as React from "react";
+
+import { hapi } from "shared/hapi/release";
+import { IChartUpdateInfo, IRelease } from "shared/types";
+
+ClarityIcons.addIcons(exclamationTriangleIcon, checkCircleIcon, circleArrowIcon);
+
+interface IChartInfoProps {
+  app: IRelease;
+}
+
+function getUpdateInfo(updateInfo: IChartUpdateInfo, chartMetadata: hapi.chart.IMetadata) {
+  if (updateInfo.error) {
+    // Unable to get info, return error
+    return (
+      <div className="color-icon-danger">
+        <CdsIcon shape="exclamation-triange" size="md" solid={true} /> Update check failed.{" "}
+        {updateInfo.error.message}
+      </div>
+    );
+  }
+  if (updateInfo.upToDate) {
+    // App is up to date
+    return (
+      <div className="color-icon-success">
+        <CdsIcon shape="check-circle" size="md" solid={true} /> Up to date
+      </div>
+    );
+  }
+  if (chartMetadata.appVersion !== updateInfo.appLatestVersion) {
+    // There is a new application version
+    return (
+      <div className="color-icon-info">
+        <CdsIcon shape="circle-arrow" size="md" solid={true} /> A new version for{" "}
+        {chartMetadata.name} is available: {updateInfo.appLatestVersion}.
```

Comment:
Can we make `updateInfo.appLatestVersion` a link to the README file of that version. I think it's an interesting information.

Reply:
I wouldn't do so for two reasons:
 - READMEs are mostly identical between versions, the new README won't likely have new information.
 - (more important) The README is shown in the catalog view for that chart. There the action item is "Deploy", that may confuse the user that would deploy a new chart rather than upgrading the existing one. What we had before was a link to the upgrade form but since there is a button in the header I thought it was not that helpful.